dogs.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  <title>Adoptadog</title>
+</head>
+
+<body>
+
+  <body>
+
+    <header class="header">
+      <div class ="header__logo">
+        <img src="images/adoptadog-logo.svg" alt="adoptadog logo">
+
+        <h1>Seattle’s premiere dog adoption agency</h1>
+      </div>
+
+      <div class="header__cart">
+
+        <img src="images/cart-icon.svg" alt="shopping cart icon">
+
+        <p>$1234.56</p>
+
+        <button type="button" name="Checkout">Checkout</button>
+
+      </div>
+
+
+      <nav class="container">
+          <a href="./index.html">Home / </a>
+          <a href="./dogs.html">Dogs / </a>
+        <a href="./blog.html">Blog </a>
+      </nav>
+
+    </header>
+
+  <section class="container">
+
+    <div class="">
```

Comment:
Your structure is looking great. You'll definitely want to have some sort of container for your dog tiles. I would recommend changing this to "dog-tiles-container" or something along those lines. Additionally, you'll want a class to style the individual dog tiles. You can achieve the grid appearance by using css grid, or flexbox. Here is a great example on using flexbox for this: [https://css-tricks.com/dont-overthink-flexbox-grids/](https://css-tricks.com/dont-overthink-flexbox-grids/)